lonestar/matrixcompletion/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ if(CMAKE_COMPILER_IS_GNUCC)
   target_compile_options(matrixCompletion PRIVATE -ffast-math)
 endif()
 
-find_package(Eigen CONFIG)
```

Comment:
Great question. The short answer is that I tested it without the additional option and it worked. After looking it up, the longer answer is that the default searches first for a `FindEigen3.cmake` module as it would in the MODULE version of find_package, then it does the usual CONFIG version of find_package. Since we don't have a `FindEigen3.cmake` it still does the right thing. We may just want to set `CMAKE_FIND_PACKAGE_PREFER_CONFIG=ON` in our root CMakeLists to not have it bother with the extra search.

Reply:
I've been reflexively putting `CONFIG` for any new-style packages (i.e., `target_link_libraries(... my::lib)`) if only to not deal with the legacy behavior of `add_compile_flags`/`include_directories`/.... I don't think this matters in the limit because it is unlikely that there are any FindEigen3.cmake files floating around anymore. So leaving it as is seems ok to me.

I do have a small preference to not set the global property if only to try to keep it the responsibility of whoever is adding `find_packages` to do the appropriate thing.

Reply:
Okay, I've re-added the CONFIG. Either way is really fine with me.